config/autoload/doctrine.local.php.dist

```diff
@@ -1,18 +0,0 @@
-<?php
-
-return [
-    'doctrine' => [
-        'connection' => [
-            'orm_default' => [
-                'driverClass' =>'Doctrine\DBAL\Driver\PDOMySql\Driver',
-                'params' => [
-                    'host'     => 'localhost',
-                    'port'     => '3306',
-                    'user'     => 'root',
-                    'password'     => '',
-                    'dbname'   => 'learnzf2',
-                ],
-            ],
-        ],
-    ]
-];
```

Comment:
why you remove this? please keep it, do copy instead : 
```
cp config/autoload/doctrine.local.php.dist config/autoload/doctrine.local.php
```
and modify <code>config/autoload/doctrine.local.php</code> based on your local env